ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import getpass
+import os
+import platform
+
+from ros2pkg.api.create import create_folder
+from ros2pkg.api.create import create_template_file
+from ros2pkg.verb import VerbExtension
+
+
+class CreateVerb(VerbExtension):
+    """Create a new ROS2 package."""
+
+    def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            'package_name',
+            help='The package name')
+        parser.add_argument(
+            '--destination-directory',
+            default=os.path.realpath(os.getcwd()),
+            help='Directory where to create the package')
+        parser.add_argument(
+            '--build-tool',
+            default='ament_cmake',
+            choices=['cmake', 'ament_cmake'],
+            help='Which build tool to use')
+        parser.add_argument(
+            '--dependencies',
+            nargs='+',
+            default=[],
+            help='list of dependencies')
+        parser.add_argument(
+            '--maintainer-email',
+            default=getpass.getuser() + '@' + platform.uname()[1] + '.local',
+            help='email address of the maintainer of this package'),
+        parser.add_argument(
+            '--maintainer-name',
+            default=getpass.getuser(),
+            help='name of the maintainer of this package'),
+        parser.add_argument(
+            '--create-cpp-exe',
+            nargs='?',
+            const=True,
+            required=False,
+            help='create an empty cpp executable')
+        parser.add_argument(
+            '--cpp-exe-name',
+            nargs='+',
+            help='name of the empty cpp executable')
+        parser.add_argument(
+            '--create-cpp-library',
+            nargs='?',
+            const=True,
+            required=False,
+            help='create an empty cpp library')
+        parser.add_argument(
+            '--cpp-exe-library',
+            nargs='+',
+            help='name of the empty cpp library')
+
+    def main(self, *, args):
+        package_name = args.package_name
+        destination_directory = args.destination_directory
+        build_tool = args.build_tool
+        dependencies = args.dependencies
+        maintainer_email = args.maintainer_email
+        maintainer_name = args.maintainer_name
```

Comment:
All these local variables seem unnecessary. Just using the `args.*` variables should be ok or is there a reason not to?